StudentDetails.java

```diff
@@ -0,0 +1,155 @@
+import java.util.*;
+import java.util.Scanner;
+
+class StudentData {
+    private int sno;
```

Comment:
What is __sno__? For you it is totally clear, but avoid abbreviations if possible. A new member to your team will have no clue that __sno__ is probably the __student number__. So just use __studentNumber__ instead of __sno__.